scripts/adlsgen2setup.py

```diff
@@ -135,7 +135,8 @@ async def create_or_get_group(self, group_name: str):
                 logging.info(f"Could not find group {group_name}, creating...")
                 group = {
                     "displayName": group_name,
-                    "groupTypes": ["Unified"],
+                    "mailEnabled": False,  # Set mailEnabled to False
```

Comment:
Do you know why groupTypes couldn't be Unified? Is this particular to your tenant, or all tenants?

Reply:
I suspect this is a tenant-specific issue - "Unified" happened to work for me

Reply:
@john-carroll-sw Since this is a tenant-specific fix, could you instead put a comment here that says 
"If Unified does not work for you, then you may need the following settings instead- "mailEnabled": False, "mailNickname": group_name"
We're not sure which situation is more common.